app/src/main/java/com/cornellappdev/android/eatery/ui/navigation/MainTabbedNavigation.kt

```diff
@@ -362,7 +362,13 @@ fun SetupNavHost(
             }) {
             FavoritesScreen(onEateryClick = {
                 navController.navigate("${Routes.EATERY_DETAIL.route}/${it.id}")
-            })
+            },onSearchClick = {
+                FirstTimeShown.firstTimeShown = false
+                navController.navigate(Routes.SEARCH.route)
+            },
+                onBackClick = {
+                    navController.popBackStack()
+                })
```

Comment:
Formatting is sus >:(

I think Zach fixed this instance on his PR but plz make sure to format code with ctrl alt L :-)

Reply:
done!